### PR TITLE
Fix: Changed behaviour of esc_sql() in WordPress 4.8.3

### LINKED ISSUE #42: Since WordPress 4.8.3 export creates bad sql dump (% characters replaced by some random hash)
See details:
[Changed behaviour of esc_sql() in WordPress 4.8.3](https://make.wordpress.org/core/2017/10/31/changed-behaviour-of-esc_sql-in-wordpress-4-8-3/)

### PR DESCRIPTION
**Pull request by mullnerz**
Fixes #42

### CODE REVIEW

**Comment by mullnerz** [comment]
Hello @szabacsik

File: src/Search_Replace_Command.php [hunk 1]
```diff
@@ -547,7 +547,14 @@ private function write_sql_row_fields( $table, $rows ) {
 			if( ( $index % $export_insert_size == 0 && $index > 0 ) || $index == $count ) {
 				$sql .= ";\n";
 
-				$sql = $wpdb->prepare( $sql, array_values( $values ) );
+				if( method_exists( $wpdb, 'remove_placeholder_escape' ) ) {
+					// since 4.8.3
+					$sql = $wpdb->remove_placeholder_escape( $wpdb->prepare( $sql, array_values( $values ) ) );
```
Comment (danielbachhuber) [inline src/Search_Replace_Command.php]: Is there a reason we can't internalize `remove_placeholder_escape` and use it with earlier WP versions?

Comment (mullnerz) [reply src/Search_Replace_Command.php]: You don't need it for earlier WP versions. WP internal behavior changed in 4.8.3 and the method remove_placeholder_escape was provided at the same time to help plugin developers:
https://make.wordpress.org/core/2017/10/31/changed-behaviour-of-esc_sql-in-wordpress-4-8-3/

**Comment by danielbachhuber** [comment]
Requesting a second pair of eyes from @gitlost

**Pull request merged**